cogboard-app/src/main/kotlin/com/cognifide/cogboard/logStorage/LogStorage.kt

```diff
@@ -155,7 +156,7 @@ class LogStorage(
     }
 
     /** Downloads new logs and inserts the to the database. Returns the number of inserted logs. */
-    private fun downloadInsertLogs(seq: Long, skipFirstLines: Long? = null): Long {
+    private fun downloadInsertLogs(seq: Long, skipFirstLines: Long? = null): List<Log> {
```

Comment:
możemy to jakoś ładniej rozbić i nazwać? ta funkcja robi bardzo dużo i jeszcze nazwa nie obejmuje wszystkiego, co się dzieje w środku: pobieranie logów, parsowanie, filtrowanie i na koniec jeszcze zapisywanie

Reply:
Podzielono na `downloadFilterLogs` i `insertLogs`.